lib/rubocop/rbs/version.rb

```diff
@@ -2,6 +2,9 @@
 
 module RuboCop
   module RBS
-    VERSION = '1.4.2'
+    # This module holds the RuboCop on RBS version information.
+    module Version
+      STRING = '1.4.2'
```

Comment:
Why need this change?

Reply:
This change was not necessary so it was reverted.